_test/test_sqw_pixels/test_mask.m

```diff
@@ -38,6 +38,9 @@
 
             % 2D case setup
             obj.sqw_2d = sqw(obj.sqw_2d_file_path, 'file_backed', false);
+            % are signal and error in the sample object incorrect?            
```

Comment:
Not clear what this comment is for

Reply:
There is the operation, below this line, which recalculates bin data. Apparently, its `s,e` are different from the one, stored in the file. I did the checks and believe, the code is correct, so reference data are apparently incorrect. Worth knowing this.